src/effects/Reverse.cpp

```diff
@@ -78,7 +78,7 @@ bool EffectReverse::Process(EffectInstance &, EffectSettings &)
    int count = 0;
```

Comment:
Typo in commit message: `Lower the loop over channels in EffectProgress` instead of `Lower the loop over channels in EffectReverse`